user/deployment/snaps.md

```diff
@@ -1,38 +1,52 @@
 ---
-title: Ubuntu Snap Store
+title: Snap Store
 layout: en
 permalink: /user/deployment/snaps/
 ---
 
-Travis CI can automate the continuous delivery to the [Snaps](https://snapcraft.io) Store.
+Travis CI can automatically upload and release your snap to the [snap store](https://snapcraft.io) after a successful build.
```

Comment:
Snap Store is capitalised. I'd say release your project or release your app.

Reply:
I'm a tad worried that changing "snap" to "app" will make folks assume the deployer will build their snap for them, but done.